src/controllers/userController.js

```diff
@@ -0,0 +1,32 @@
+import { Login, Logout, Refresh } from "../service/userService.js";
```

Comment:
Лучше импортировать модуль полностью, чтобы видеть, какому конкретно сервису принадлежит метод.
```suggestion
import UserService from "../service/userService.js";
```